### PR TITLE
Index cluster.id, node.id, and node.name in elasticsearch/node_stats metricset

### PR DESCRIPTION
**Pull request by ycombinator**
This PR teaches the `elasticsearch/node_stats` metricset to index the Elasticsearch `cluster_uuid` as the module-level `cluster.id` field, as well as the node ID and node `name` fields as the metricset-level `node.id` and `node.name` fields, respectively.

### CODE REVIEW

File: metricbeat/module/elasticsearch/node_stats/data.go [hunk 1]
```diff
@@ -140,6 +141,10 @@ func eventsMapping(r mb.ReporterV2, content []byte) error {
 			errs = append(errs, event.Error)
 		}
 
+		name, _ := event.MetricSetFields.GetValue("name")
+		event.ModuleFields.Put("node.name", name.(string))
```
Comment (ruflin) [inline metricbeat/module/elasticsearch/node_stats/data.go]: We should probably check the type conversion here? Also what happens if above returns and error and `name` does not exist? Will `name` be nil?

**Comment by ycombinator** [comment]
CI test failure is valid. Fixing...

File: metricbeat/module/elasticsearch/node_stats/data.go (hunk 1 above)
Comment (ruflin) [reply metricbeat/module/elasticsearch/node_stats/data.go]: I see. Agree that it should not break here but what if someone modifies the other part of the code and does not thing about this part? I normally prefer to be on the safe side.

File: metricbeat/module/elasticsearch/node_stats/_meta/fields.yml [hunk 2]
```diff
@@ -1,158 +1,167 @@
...
+- name: node
   type: group
   description: >
-    node_stats
+    Node
   release: beta
   fields:
-    - name: indices
-      type: group
+    - name: id
```
Comment (ruflin) [inline metricbeat/module/elasticsearch/node_stats/_meta/fields.yml]: I would assume this is now a duplicated field as it's also in `node` metricset? I general I would recommend to configure "shared" fields on the module level.

**Comment by ruflin** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
@ruflin Moved node fields to module level. Ready for your 👀 again. Thanks!

File: metricbeat/module/elasticsearch/node_stats/_meta/fields.yml [hunk 3]
```diff
@@ -1,158 +1,163 @@
-- name: node.stats
+- name: node
   type: group
   description: >
-    node_stats
+    Node
   release: beta
   fields:
-    - name: indices
+    - name: stats
```
Comment (ruflin) [inline metricbeat/module/elasticsearch/node_stats/_meta/fields.yml]: Not sure why you introduce the mapping here (was it for the other fields)? In the end both should work.

Comment (ycombinator) [reply metricbeat/module/elasticsearch/node_stats/_meta/fields.yml]: Ugh, this change should've been reverted when I moved the node fields to the module level. Sorry about that. Fixing now...

Comment (ycombinator) [reply metricbeat/module/elasticsearch/node_stats/_meta/fields.yml]: Reverted in b5912d88dc038d9adef72ff8f8ee13e782c37c0c.

**Pull request merged**